website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,30 @@
+## Content
```

Comment:
I am not sure this is the content that I would expect in this section. I would imagine some guidance about how and where to use this component, its relation with the `Modal` and `Flyout`,  some dos/donts, etc. This dry content sounds more suited for the "anatomy" section.